### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by jquense**
i still need to tweak the actual cli a bit b/c its sort of odd to do `format --check` for linting since eslint now runs. I think it's worth having a `lint` and `format` command where the latter doesn't output none fixable eslint problems and the former shows everything

### CODE REVIEW

File: packages/pedantic/Linter.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      return content;
+    }
+    const { results } = this.cli.executeOnText(content, filePath);
+    if (!results.length) return content;
+
+    const [result] = results;
+
+    // filter out fixed errors
+    if (this.fix && (result.fixableErrorCount || result.fixableWarningCount)) {
+      console.log('herer!');
```
Comment (taion) [inline packages/pedantic/Linter.js]: remove

File: packages/pedantic/Linter.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      this.fixErrorCount += result.fixableErrorCount;
```
Comment (taion) [inline packages/pedantic/Linter.js]: how do these get used?

**Comment by jquense** [comment]
done!

File: packages/pedantic/lint.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const format = require('./lib');
+
+exports.command = 'lint <patterns..>';
+
+exports.describe = 'Lint files use eslint, prettier and import sort';
```
Comment (taion) [inline packages/pedantic/lint.js]:
```suggestion
exports.describe = 'Lint files use ESLint, Prettier and import-sort';
```

**Pull request merged**